hives.go

```diff
@@ -50,4 +50,5 @@ import (
 	_ "github.com/muesli/beehive/bees/twiliobee"
 	_ "github.com/muesli/beehive/bees/twitterbee"
 	_ "github.com/muesli/beehive/bees/webbee"
+	_ "github.com/muesli/beehive/bees/cricketbee"
```

Comment:
Those imports are supposed to be sorted alphabetically. Simply run go fmt on all your code - it will take care of it for you!